src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -204,6 +204,7 @@ private object ProcessValue(object obj, int depth)
             else
             {
                 TypeInfo t = obj.GetType().GetTypeInfo();
+                WriteVerbose(StringUtil.Format(UtilityCommonStrings.ConvertToJsonProcessValueVerboseMessage, t.Name, depth));
```

Comment:
This doesn't seem like the right fix to me.  The verbose message was removed for a reason.  If a test depended on the verbose message, then it (the test) should be removed or re-written.

Reply:
You can just temporarily disable the test until #8593 is complete.  It seems wrong to revert a previously approved change without good cause.

Reply:
It may be best to use an [InternalTestHook](https://github.com/PowerShell/PowerShell/blob/d80154430d600a52a1d1e3a9f3ecf032b36185d4/src/System.Management.Automation/engine/Utils.cs#L1796) to get the cmdlet to write the VerboseMessage if the message is not useful outside of the test